app/steps/ui/deceased/allchildrenover18/index.js

```diff
@@ -1,12 +1,33 @@
 'use strict';
 
 const ValidationStep = require('app/core/steps/ValidationStep');
+const FormatName = require('app/utils/FormatName');
+const content = require('app/resources/en/translation/deceased/anychildren');
 
 class AllChildrenOver18 extends ValidationStep {
 
     static getUrl() {
         return '/all-children-over-18';
     }
+
+    getContextData(req) {
+        const ctx = super.getContextData(req);
+        const formdata = req.session.form;
+        ctx.deceasedName = FormatName.format(formdata.deceased);
```

Comment:
Do we not need to add to the action method, 
delete ctx.deceasedName

Reply:
Yes, definitely.